src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Fixtures/TestApplicationFactory.cs

```diff
@@ -52,6 +52,11 @@ protected override IHostBuilder CreateHostBuilder()
                     builder
                         .AddFilter<ConsoleLoggerProvider>("Default", LogLevel.Warning)
                         .AddFilter<ConsoleLoggerProvider>("Microsoft", LogLevel.Warning);
+
+                    // Uncomment to add SQL logging to the debug console.
+                    // .AddFilter<ConsoleLoggerProvider>((category, level) =>
+                    //     category == DbLoggerCategory.Database.Command.Name
+                    //     && level == LogLevel.Information);
```

Comment:
are we leaving this here on purpose?

Reply:
I'd like to, as currently I'm finding it very useful to just be able to include this again when I need it.  I did try to control with configuration, but it was getting messy!